exercicios/para-casa/sistema_de_alunas.py

```diff
@@ -37,17 +37,38 @@ def obter_opcao():
     
 def incluir_nova_aluna():
     pass
-    #TODO - Implentar a função
+    from dataset_alunas import dataset
+    nome_completo = input("\nDigite o nome completo da nova aluna: ")
+    sobrenome = input("Digite o sobrenome da aluna: ")
+    turma = input("Digite a turma da aluna: ")
+    
+
+
     
 def consultar_lista_alunas():
     pass
-    #TODO - Implentar a função
+    if  dataset :
+        print("\n Lista de Alunas Cadastradas:") 
+    for idx, aluna in enumerate(dataset ):
+            print(f"{idx}. {aluna['nome']} {aluna['sobrenome']} - Turma: {aluna['turma']}")
```

Comment:
Implementar de forma mais simples ajuda na legibilidade do código. Colocar variáveis com nomes intuitivos ajuda no processo:

**Sugestão:**
```suggestion
    for nome, sobrenome in dataset.keys():
        print(f'- {nome} {sobrenome}')
```